ui/raidboss/data/06-ew/raid/p6s.ts

```diff
@@ -471,6 +473,34 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
+    {
+      id: 'P6S Transmission',
+      type: 'GainsEffect',
+      // CF3 Chelomorph (Wing icon - cleave behind player)
+      // D48 Glossomorph (Snake icon - cleave in front of player)
+      netRegex: NetRegexes.gainsEffect({ effectId: ['CF3', 'D48'] }),
+      condition: Conditions.targetIsYou(),
+      preRun: (data, matches) => {
+        // 1st transmission has 11s duration, 2nd has 25s duration
+        // in either case, trigger should fire 3s before debuff expires
+        const duration = parseFloat(matches.duration);
+        data.transmissionAlertDelay = duration > 3 ? duration - 3 : 0;
+      },
+      delaySeconds: (data) => {
+        return data.transmissionAlertDelay;
+      },
```

Comment:
Not sure why you need a variable here.  I think you can do this just in the function itself.

```suggestion
      delaySeconds: (data, matches) => {
        // 1st transmission has 11s duration, 2nd has 25s duration
        // in either case, trigger should fire 3s before debuff expires
        return parseFloat(matches.duration) - 3;
      },
```

Reply:
Wasn't sure at the time if I would also be using it to manipulate the timing on the cachexia2 call to prevent too many calls from firing at the same time, but I think it should be okay.